### DISCUSSION TITLE
Android - Can't update or connect

### DISCUSSION THREAD

**Discussion by lukebarone** [rustdesk/rustdesk#9279] [General]
### Bug Description

Samsung S20 FE with Android 13. When I attempt to update through F-Droid, it claims my phone is not compatible. No update appears from the Play Store.

Main issue is I cannot enter a 2FA code for a new installation. I can enter the password, but not the extra code.

### How to Reproduce

Install 1.1.10-1

Attempt to update via Play Store (none available) or F-Droid (Not compatible)

### Expected Behavior

Updated client to use new features

### Operating system(s) on local side and remote side

Android 13 -> Windows

### RustDesk Version(s) on local side and remote side

1.1.10-1 -> 1.3.0

### Screenshots

![Screenshot_20240906_175235](https://github.com/user-attachments/assets/2c39cdcd-a338-4ad5-9e99-2f1cd1ae95cc)

### Additional Context

I do not have a Pro account, but I self host my ID and Relay server

**Comment by rksdutt** [comment]
Open Windows command promtp
Check system time
time /t

2FA issues by ensuring correct time settings
verifying configuration, updating apps, using alternative methods
also contacting support if problems persist.

**Comment by basilgello** [comment]
@lukebarone What is F-Droid version and how does F-Droid's landing page for RustDesk app look like? Especially version list at the bottom?

**Reply by lukebarone** [reply]
![Screenshot_20240909_062906_F-Droid](https://github.com/user-attachments/assets/310e06c3-63dc-4d70-b419-bb280d9d5b9c)

**Reply by basilgello** [reply]
Did you install 1.1.10-1 from F-Droid? What happens if you uninstall the app and try installing it via F-Droid? (this will kill all Rustdesk saved sessions and settings!) Also, can you install https://f-droid.org/en/packages/com.kgurgul.cpuinfo/ and show its output prior to uninstalling RustDesk?

**Reply by basilgello** [reply]
And please show version of F-Droid client itself: Settings -> About F-Droid

**Comment by lukebarone** [comment]
F-Droid version 1.20.0
Yes, installed from F-Droid.
Uninstalled, reinstalled and it chose 1.3.0.

I think this was a me-issue - it's possible I didn't install it from
F-Droid (for some reason, my launcher says it came from Google Play). My
remote computer is offline, so I'll have to confirm tonight.

Thank you for your help so far.
